client/src/_shared/Header.js

```diff
@@ -90,6 +91,7 @@ export function Header() {
                 className={
                   location.pathname.includes('/attendance') ? 'underline' : ''
                 }
+                id={'dashboard-tab'}
```

Comment:
You cannot have the same id twice in a form

Reply:
fixed